ui/app/services/version.js

```diff
@@ -32,20 +42,29 @@ export default class VersionService extends Service {
     return this.features.includes('Control Groups');
   }
 
-  get isEnterprise() {
-    if (!this.version) return false;
-    return this.version.includes('+');
-  }
-
-  get isOSS() {
-    return !this.isEnterprise;
+  get versionDisplay() {
+    if (!this.version) {
+      return '';
+    }
+    return this.isEnterprise ? `v${this.version.slice(0, this.version.indexOf('+'))}` : `v${this.version}`;
   }
 
-  @task
+  @task({ drop: true })
   *getVersion() {
     if (this.version) return;
-    const response = yield this.store.adapterFor('cluster').sealStatus();
-    this.version = response.version;
+    const response = yield this.store.adapterFor('cluster').fetchVersion();
+    this.version = response.data.version;
+  }
+
+  @task({ drop: true })
+  *getType() {
+    if (this.type !== null) return;
+    const response = yield this.store.adapterFor('cluster').health();
+    if (typeof response.enterprise === 'boolean') {
```

Comment:
`enterprise` is a key that will be added to the `sys/health` endpoint. Until then we fallback to checking for `license`